.github/workflows/fireperf-e2e.yml

```diff
@@ -0,0 +1,107 @@
+          token: ${{ secrets.GOOGLE_OSS_BOT_TOKEN }}
+          path: firebase-android-buildtools
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          distribution: temurin
+          cache: gradle
+      - name: Set up NDK 21.4.7075529
+        run: |
+          ANDROID_ROOT=/usr/local/lib/android
+          ANDROID_SDK_ROOT=${ANDROID_ROOT}/sdk
+          ANDROID_NDK_ROOT=${ANDROID_SDK_ROOT}/ndk-bundle
+          SDKMANAGER=${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/sdkmanager
+          echo "y" | $SDKMANAGER "ndk;21.4.7075529"
+          ln -sfn ${ANDROID_SDK_ROOT}/ndk/21.4.7075529 ${ANDROID_NDK_ROOT}
+          echo "ANDROID_NDK_HOME=${ANDROID_NDK_ROOT}" >> $GITHUB_ENV
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Set up fireci
+        run: pip3 install -e ci/fireci
+      - uses: google-github-actions/auth@v0
+        with:
+          credentials_json: ${{ secrets.GCP_SERVICE_ACCOUNT }}
+      - uses: google-github-actions/setup-gcloud@v0
+      - name: Add google-services.json
+        run: echo $PERF_E2E_GOOGLE_SERVICES | base64 -d > google-services.json
+      - name: Run fireperf end-to-end tests
+        run: |
+          fireci fireperf_e2e_test \
+            --plugin_repo_dir=firebase-android-buildtools \
+            --target_environment=${{ matrix.environment }}
+      - name: Notify developers upon failures
+        if: ${{ failure() }}
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const owner = context.repo.owner;
+            const repo = context.repo.repo;
+            const commit = context.sha;
+            const run = context.runId;
+            const url = `https://github.com/${owner}/${repo}/actions/runs/${run}`;
+
+            const datetime = (new Date()).toLocaleString('en-US', {
+              timeZone: 'America/Los_Angeles',
+              dateStyle: 'medium',
+              timeStyle: 'long',
+            });
+
+            const text =
+            `Failed on commit ${commit} at ${datetime}.
+
+            ${url}`;
+
+            const { data: issues } = await github.rest.issues.listForRepo({
+              owner: owner,
+              repo: repo,
+              state: 'open',
+              labels: 'fireperf-e2e-tests'
+            });
+
+            if (issues.length) {
+              github.rest.issues.createComment({
+                owner: owner,
+                repo: repo,
+                issue_number: issues[0].number,
+                body: text,
+              });
+            } else {
+              github.rest.issues.create({
+                owner: owner,
+                repo: repo,
+                title: 'FirePerf E2E Test Failures',
+                body: text,
+                labels: ['fireperf-e2e-tests'],
+                assignees: ['jeremyjiang-dev', 'leotianlizhan', 'visumickey']
```

Comment:
fyi @jeremyjiang-dev @leotianlizhan @visumickey does that look good to you?

Reply:
Could you also add @raymondlam ? LGTM

Reply:
Done.